src/Lit/Experimental.fs

```diff
@@ -0,0 +1,76 @@
+[<Experimental("This is an exploration space, anything can be added or removed in this module at any point")>]
+module Lit.Experimental
+
+
+open Fable.Core
+open Fable.Core.JsInterop
+open Lit
+
+[<AttachMembers>]
+type StateController<'T>(host, initial: 'T) =
+    inherit ReactiveController(host)
+    let mutable state = initial
+
+    member _.Value = state
+
+    member _.SetState(value: 'T) =
+        state <- value
+        host.requestUpdate ()
+
+[<AttachMembers>]
+type TwoArgsCtrl(host, initial: int, secondial: int) =
+    inherit ReactiveController(host)
+
+    member val Values = (initial, secondial) with get, set
+
+    member this.updateValues(initial, secondial) =
+        this.Values <- (initial, secondial)
+        host.requestUpdate ()
+
+
+type Controllers =
+
+    static member inline GetController<'T>([<System.ParamArray>] args: obj array) : 'T =
+        jsThis?getOrAddController (jsConstructor<'T>, args)
+
+    static member inline GetProperty(propName: string, ?initial: 'T) : Types.RefValue<'T> =
+        jsThis?getOrAddProperty (propName, initial)
```

Comment:
As I was saying in a comment above, Context would be a better name for this rather than controllers

This plays on the idea that Function Based LitElements have a class that can store either properties/controllers or similar things so rather than have an initialize function we should be able to grab them from the backing class of those decorated functions.

This is kind of crafted for my own experiment but I think we could somehow make it work with your existing LitElement functions without taking away the existing support for the init function this no breaking changes + a way to progress forward